ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -176,14 +176,17 @@ enum Option {
       DEFAULT,
 
       /** Use ReadIndex (see Raft Paper section 6.4). Maintains linearizability */
-      LINEARIZABLE
+      LINEARIZABLE,
+
+      /** Use Leader lease. Efficient than ReadIndex if clock drift is bounded */
+      LEASE
```

Comment:
It seems that both DEFAULT and LINEARIZABLE are talking about consistency issues. Lease and ReadIndex are implementation techniques for implementing LINEARIZABLE, so it seems inappropriate to mention lease alone alongside the other two options?

Reply:
You are right. DEFAULT / LINEARIZABLE / LEASE sound vague to the terminal users. We should consider renaming these options.

Reply:
Is there a good reason to use  LINEARIZABLE not LEASE? 
- If yes, we could have a separated configuration to enable/disable LeaderLease.
- If not, we could just have LINEARIZABLE for ReadIndex + LeaderLease.

+1 Patch looks good other that this.  Thanks @SzyWilliam for working on this!